packages/hosted-credit-card-integration/src/HostedCreditCardPaymentMethod.tsx

```diff
@@ -64,11 +64,17 @@ const HostedCreditCardPaymentMethod: FunctionComponent<PaymentMethodProps> = ({
             const isInstrumentCardCodeRequired = selectedInstrument
                 ? isInstrumentCardCodeRequiredProp(selectedInstrument, method)
                 : false;
-            const styleContainerId = selectedInstrument
-                ? isInstrumentCardCodeRequired
-                    ? getHostedFieldId('ccCvv')
-                    : undefined
-                : getHostedFieldId('ccNumber');
+            let styleContainerId;
```

Comment:
Can we please update tests to cover this change.

Reply:
@animesh1987 yeah, forgot about it. I haven't refactored whole test file, due to small estimation of this ticket. We can create a separate ticket for the refactoring to RTL